fab/fabfile.py

```diff
@@ -462,8 +453,8 @@ def update_code(use_current_release=False):
 
     with cd(env.code_root if not use_current_release else env.code_current):
         sudo('git remote prune origin')
-        sudo('git fetch')
-        sudo("git submodule foreach 'git fetch'")
+        sudo('git fetch origin {}'.format(env.code_branch))
+        sudo("git submodule foreach 'git fetch origin {}'".format(env.code_branch))
```

Comment:
ah yea you're right, i actually don't think you'd want to fetch the `code_branch` for the submodules at all. i guess that means you'd have to pull down every branch just in case that hotfix refers to something that isn't on master